.additional_bashrc.sh

```diff
@@ -10,6 +10,8 @@ NODE_CONTAINER=$(sudo docker ps -f "name=${DOCKER_COMPOSE_PROJECT}_node_1" --for
 alias node='sudo docker exec -u1000 -w $(pwd) -it ${NODE_CONTAINER} node'
 alias npm='sudo docker exec -u1000 -w $(pwd) -it ${NODE_CONTAINER} npm'
 alias yarn='sudo docker exec -u1000 -w $(pwd) -it ${NODE_CONTAINER} yarn'
+alias node_exec='sudo docker exec -u1000 -w $(pwd) -it ${NODE_CONTAINER}'
```

Comment:
needs Documentation

Reply:
Please see: https://github.com/pluswerk/php-dev/pull/25

Reply:
I'll add documentation

- regarding #25 - there we encountered some minor problems, the group that node is using should be www-data, else the nginx/apache might have difficulties handling builded files that is being processed afterwards by php-fpm

Reply:
> I'll add documentation
> 
> * regarding #25 - there we encountered some minor problems, the group that node is using should be www-data, else the nginx/apache might have difficulties handling builded files that is being processed afterwards by php-fpm

see #34